docs/first-time-build.md

```diff
@@ -68,7 +68,7 @@ make init-package
 > ```
 > It is likely that you've forgotten to setup access to Iron Bank _or_ that your credentials have changed. In either case, you should go back through the steps to [Log into Iron Bank](#2-log-into-iron-bank) & try to build again!
 
-Assuming everything works out, you should see a shiny new `zarf-init.tar.zst` in your `./build` directory.
+Assuming everything works out, you should see a shiny new `zarf-init-<arch>.tar.zst` in your `./build` directory.
```

Comment:
There were a lot of updates to the docs and it seems like this was the only place that did <arch> instead of just writing out the two different init filenames fully.

I'm currently liking this way more but I would just ask we be consistent with however we do it.

Reply:
I'll try to sift back through those in a bit to standardize them